templates/webapi/test/infopanel.html.ep

```diff
@@ -44,14 +44,13 @@
                         Result: <b><%= $job->result %></b>\
                     % }
                     % else {
+                        % my $gru_tasks_items = $self->gru_tasks_items($job);
                         % my $state_detail = '';
-                        % if (my @gru_deps = $job->gru_dependencies) {
-                            % $state_detail = join(';', map {
-                            %   'id: ' . $_->id . ', name: ' . $_->taskname
-                            %  } map { $_->gru_task // () } @gru_deps);
-                            % $state_detail = ", waiting for background tasks ($state_detail)";
-                        % }
-                        State: <b><%= $job->state %></b><%= $state_detail %>\
+                        % $state_detail = join(';', map {
+                        %   $_->{href} ? link_to($_->{label} => $_->{href}) : $_->{label}
+                        % } @$gru_tasks_items);
+                        % $state_detail = qq{, waiting for background tasks ($state_detail)};
+                        State: <b><%= $job->state %></b><%== $state_detail %>\
```

Comment:
TODO: I'm not sure about this without any checks to the $gru_tasks_items, should I check for not empty and if it is skip the detail creation?